xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-macro/src/main/java/org/xwiki/livedata/internal/macro/LiveDataMacro.java

```diff
@@ -116,9 +123,11 @@ public LiveDataMacro()
         // Load the JavaScript code of the Live Data widget.
         Map<String, Object> skinExtensionParameters = Collections.singletonMap("forceSkinAction", Boolean.TRUE);
         this.jsfx.use("uicomponents/widgets/liveData.js", skinExtensionParameters);
+        // Load the CSS code of the Live Data widget.
+        this.ssfx.use("uicomponents/widgets/liveData.css");
```

Comment:
The loading styles should be generic so I would keep them in the skin. Moreover, I don't see what else we could add to ``liveData.css`` in the future to justify its existence beyond the live data loading animation. The minimum height is an issue that needs to be fixed for any loading block: it doesn't make sense to use a spinner icon without making sure the container has enough height to show it, so IMO this needs to be fixed in [``misc.less``](https://github.com/xwiki/xwiki-platform/blob/master/xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/less/misc.less#L327). Basically we need to drop the custom ``.tab-content > .tab-pane.loading`` fix and make it generic for any ``.loading``.